build/NUCLEO_F303K8-device.mk

```diff
@@ -0,0 +1,44 @@
+# Copyright 2015 Larry Littlefield (https://developer.mbed.org/users/svkatielee/)
```

Comment:
It would be helpful if you placed your name here so that people know who added the support for this device without needing to look at git history. Thanks, Adam